.gitignore

```diff
@@ -1 +1,5 @@
 *.pyc
+/demos/.vs
+/demos/build/
```

Comment:
This seems rather developer-specific to me. I think it should be in your local `.git/info/exclude` rather than here.

Reply:
agree. customers might have other build path

Reply:
done